resources/containerd-config.toml

```diff
@@ -0,0 +1,38 @@
+# based on /run/torcx/unpack/docker/usr/share/containerd/config.toml
```

Comment:
Is this available online? If yes, could we include a link?

Reply:
I'll check